script/cibuild

```diff
@@ -57,11 +57,12 @@ trap cleanup EXIT
 export ES_PORT=${ES_PORT:-19200}
 
 es_version=${ES_VERSION:-24}
-docker_compose="docker-compose --file docker/docker-compose.es${es_version}.yml --file docker/docker-compose.cibuild.yml"
+docker_compose="docker-compose --file docker/docker-compose.es${es_version}.yml"
 
-# if [ "${CI_MODE:-}" ]; then
-#   docker_compose="${docker_compose} --no-ansi"
-# fi
+if [ "${CI_MODE:-}" ]; then
+  docker_compose="${docker_compose} --file docker/docker-compose.cibuild.yml"
+#  docker_compose="${docker_compose} --no-ansi"
```

Comment:
kill this commented out line?

Reply:
I'd like to leave it in and uncomment it once we have a more modern version of `docker-compose` out on the build hosts. The `--no-ansi` will suppress the weird control characters in the build output that normally provide colorization in a terminal.